CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## [Unreleased]
+### Changed
+- change `k3d.configureComponents` so that `null` values will delete the components from the config.
```

Comment:
This sentence is a bit misleading because it references unnamed components. How about this?

```suggestion
- change `k3d.configureComponents` so that `null` values disable single components so they won't be installed.
```